src/osd/PG.h

```diff
@@ -161,12 +161,21 @@ class PGRecoveryStats {
  *
  */
 
+class ScrubberPasskey {
```

Comment:
Add a short comment to suggest use case.
Something like
"facilitating scrub-related object access to private PG data"